custom/enikshay/duplicate_ids.py

```diff
@@ -89,3 +91,51 @@ def _get_user_info(user_ids):
         }
         for user_doc in iter_docs(CommCareUser.get_db(), user_ids)
     }
+
+
+class ReadableIdGenerator(object):
```

Comment:
@ctsims FYI here's the ID generation scheme I wrote.  Here's a file containing 20,000 consecutive IDs:
[readable_ids.txt](https://github.com/dimagi/commcare-hq/files/1564159/readable_ids.txt)
I've confirmed that there are no duplicates.